homeassistant/components/isy994/__init__.py

```diff
@@ -48,6 +51,9 @@
                 vol.Optional(
                     CONF_SENSOR_STRING, default=DEFAULT_SENSOR_STRING
                 ): cv.string,
+                vol.Optional(
+                    CONF_VAR_SENSOR_STRING, default=DEFAULT_VAR_SENSOR_STRING
```

Comment:
We don't want to extend the legacy config schema, see ADR 10.

Reply:
@shbatm At the end of this PR chain we should probably mark the legacy schema with `cv.deprecated(DOMAIN)` so users migrating from the hacs version back to core know its going away.

Reply:
@bdraco Roger. Adding to mop-up PR. Is there a specific `invalidation_version` I need to set?

@MartinHjelmare is your preference to revert this change in the mop-up PR or leave until depreciation?

Reply:
Usually I’d say revert but given the user migration path it seems not a usual case so my opinion is to leave it for a few versions so everyone can switch over cleanly.

Reply:
Sounds good.